python-sdk/tests/benchmark/wait_for_completion.sh

```diff
@@ -10,7 +10,7 @@ while [[ $complete_status -ne 0 ]] && [[ $failed_status -ne 0 ]]; do
   complete_status=$?
 done
 
-#kubectl delete job benchmark-"${GIT_HASH}" -n benchmark
+kubectl delete job benchmark-"${GIT_HASH}" -n benchmark
```

Comment:
Any idea why this was commented earlier?

Reply:
yes, when we were working on running the benchmark in CI initially, we want to check the log of pods in case of some error. but now we even delete the cluster since the flow looks stable in the current setups and we run only once in a week.